packages/database/src/TupaiaDatabase.js

```diff
@@ -108,6 +108,9 @@ export class TupaiaDatabase {
     // if a change handler is being added, this db needs a change channel - make sure it's instantiated
     this.getOrCreateChangeChannel();
     this.getChangeHandlersForCollection(collectionName)[key] = changeHandler;
+    return () => {
```

Comment:
Adding a change handler now returns a 'canceller', that you can call to stop listening

Reply:
Great idea!